paws_collector.js

```diff
@@ -122,6 +132,20 @@ class PawsCollector extends AlAwsCollector {
         return this._pawsHttpsEndpoint;
     };
 
+    reportDDMetric(name, value, tags = []) {
```

Comment:
should we have async version of it?

Reply:
I'm not really sure if there is an async version the DD method exists.

We could add a callback to the paws method, but I'm not sure it would be useful.